client/app/main/main.js

```diff
@@ -10,3 +10,21 @@ angular.module('observatory3App')
         controllerAs: 'main'
       });
   });
+
+function getEmail() {
+	var coded = "rTTPmCfUdTPB@PrTB.CT"
+    var key = "gZ8fYCPqeJjtpaFI1GBLNyuoxA7XclKd4UWSkzQ63HMThnVirbvm5R2EOD9ws0"
```

Comment:
Is this a monoalphabetic substitution cipher?
If you want to hide the email is the source code use an environment variable and get the email through a rest call, hiding with a cipher is insecure and confusing.